sponsors/views.py

```diff
@@ -23,10 +29,20 @@ class SelectSponsorshipApplicationBenefitsView(FormView):
     template_name = "sponsors/sponsorship_benefits_form.html"
 
     def get_context_data(self, *args, **kwargs):
+        programs = SponsorshipProgram.objects.all()
+        packages = SponsorshipPackage.objects.all()
+        benefits_qs = SponsorshipBenefit.objects.select_related("program")
+        capacities_met = any(
+            [
+                any([not b.has_capacity for b in benefits_qs.filter(program=p)])
+                for p in programs
+            ]
+        )
```

Comment:
Hm, I didn't get why we have to filter the `benefits_qs` by `program` to check if there's one of them without available capacity. Can't we refactor the code to:

```suggestion
        capacities_met = all([b.has_capacity for b in benefits_qs])
```

Reply:
I did this primarily since I anticipate that at some point we will have programs that are no longer active. a bit premature.